README.md

```diff
@@ -18,21 +18,58 @@ Plotly.js can be used to produce dozens of chart types and visualizations, inclu
 
 ## Table of contents
 
-* [Quick start options](#quick-start-options)
+* [Load from CDN](#load-from-cdn)
+* [Load from npm](#load-from-npm)
+* [Versioning](#versioning)
 * [Partial bundles](#partial-bundles)
 * [Alternative ways to require or build plotly.js](#alternative-ways-to-require-or-build-plotlyjs)
 * [Bugs and feature requests](#bugs-and-feature-requests)
 * [Documentation](#documentation)
 * [Contributing](#contributing)
 * [Community](#community)
-* [Versioning](#versioning)
 * [Notable contributors](#creators)
 * [Copyright and license](#copyright-and-license)
 
 ---
-## Quick start options
+## Load from CDN
```

Comment:
I would spell out "Content Delivery Network (CDN)"

Reply:
Done in https://github.com/plotly/plotly.js/pull/5705/commits/a0bd25835c3f415dece9c5cd8604663a60d7f532.